source/upgrade/upgrade_code_workflow.md

```diff
@@ -98,13 +98,33 @@ You can find all YAML files that were accessed in the last hour, for instance, w
 
 #### Node data: YAML facts
 
-The catalog preview module works by compiling catalogs for nodes and inspecting the resultant catalog. It compiles a catalog by using the existing node objects and their facts to simulate a Puppet run against that node. This means interacting with PuppetDB where all the data is stored.
+The Catalog Preview tool works by compiling catalogs for nodes and inspecting the resultant catalog. It compiles a catalog by using the existing node objects and their facts to simulate a Puppet run against that node. This means interacting with PuppetDB where all that data is stored.
```

Comment:
This change is reverting edits, please fix this line, and 103 so they're not reverting changes.